firefox_media_tests/playback/youtube/test_basic_playback.py

```diff
@@ -40,22 +28,23 @@ def test_mse_is_enabled_by_default(self):
 
     def test_video_playing_in_one_tab(self):
         with self.marionette.using_context('content'):
-            for url in self.test_urls:
+            for url in self.video_urls:
                 self.logger.info(url)
                 youtube = YouTubePuppeteer(self.marionette, url)
+                self.logger.info('Expected duration: %s' %
+                                 youtube.expected_duration)
                 youtube.deactivate_autoplay()
+
                 final_piece = 60
                 try:
                     time_left = wait_for_almost_done(youtube,
                                                      final_piece=final_piece)
                 except VideoException as e:
                     raise self.failureException(e)
-                duration = abs(youtube.player_duration) + 1
+                duration = abs(youtube.expected_duration) + 1
                 if duration > 1:
-                    self.marionette.log('Almost done: '
-                                        '%s - %s seconds left.' %
-                                        (youtube.movie_id,
-                                         time_left))
+                    self.logger.info('Almost done: %s - %s seconds left.' %
+                                     (youtube.movie_id, time_left))
                     if time_left > final_piece:
                         self.marionette.log('time_left greater than '
                                             'final_piece - %s' % time_left,
```

Comment:
I want the almost-done to appear as the test runs (self.logger) and I want the time_left message to appear in the extra log dump at the end of all tests (self.marionette.log).